common/artifacts/999_wc_artifacts_other_consumables.txt

```diff
@@ -0,0 +1,29 @@
+# Food, potions, alcohol
```

Comment:
There's a design problem with forcing one character to drink.

So, you would like to give this buff to your councillors and maybe some vassals. So, it's a lot of clicks.
I would make it so when you right-click yourself, you can invite people to drink with you getting buffs.
It will decrease the number of clicks and will be less CPU heavy for the AI.

The possibility to drink with one specific person is nice though. I would keep it and leave only for players, but I would add the event where you drink together because after giving the keg it's just silence.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103140239-bd357700-46fd-11eb-9426-16b66a38413c.png)

</details>

Reply:
You would still need to click people you want to invite so it would be still a lot of clicking plus using the decision when you invited everyone you wanted.

Reply:
plus there are already parties with carousing so it would be kinda another version of that

Reply:
> You would still need to click people you want to invite

Emm, no? You click among 4-3 options.

Reply:
> plus there are already parties with carousing so it would be kinda another version of that

Are you saying you want to include the keg opening in carousing? It sounds good.

Reply:
> > You would still need to click people you want to invite
> 
> Emm, no? You click among 4-2 options.

then it will limit who you can buff and you may want to use different brews for different chars

Reply:
> then it will limit who you can buff and you may want to use different brews for different chars

> The possibility to drink with one specific person is nice though.
> I would keep it and leave only for players.

Reply:
ok so we can add two options to share it with your wife and/or one of the advisors is that alright?

Reply:
> ok so we can add two options to share it with your wife and/or one of the advisors is that alright?

Include it into carousing or feast or both will be the best option, I think. Buffing existing button is always better than adding a new one.

Reply:
i think just adding extra options to the drinking would be the best cause adding it to carousing and feasts will be too much work for kinda useless semi-random buffs that you probably wont even want to use.

Reply:
> i think just adding extra options to the drinking would be the best cause adding it to carousing and feasts will be too much work for kinda useless semi-random buffs that you probably wont even want to use.

Mmmkay, but it would make so much sense to open kegs when you're feasting.

Reply:
i thought about it and i think the current system works the best since brews give specific buffs and you would want to use different buffs on different people and it also depends on your playstyle and character

Reply:
> i thought about it and i think the current system works the best since brews give specific buffs and you would want to use different buffs on different people and it also depends on your playstyle and character

If you didn't implement what I asked, reread what I said:
- It's less CPU heavy.
- I didn't ask you to get rid of the targeted decision.
- I asked you to share kegs with random-weighted people if you're not really interested in searching your court for the right people.